### DISCUSSION TITLE
strange bug

### DISCUSSION THREAD

**Discussion by Slykirby116** [EssentialsX/Essentials#4119] [Q&A] [answered]
so I've been playing on a minecraft server called Richdale and for some reason after a player dies they can no longer use tpa they have to use tpahere. the admins don't know what is causing this issue but it just seems to only happen when the player dies or is on the server for too long. the requests do go through and can be accepted through a normal tpa but they don't get teleported no matter how many times they try.

**Comment by JRoy** [comment, marked as answer]
Please submit a bug report if this still occurs.
